docs/generate-and-template.md

```diff
@@ -77,6 +77,7 @@ particularly useful ones are:
 - `b64enc <string>`: This base-64 encodes a string. Useful for secrets.
 - `randAlphaNum <int>`: This generates a random alphanumeric string of
   length INT
+- `env <string>`: Grabs the value of an environment variable. 
```

Comment:
"Grabs" sounds a little informal.  Could you please reword as "This retrieves the value of an environment variable."?